src/hevm/src/EVM/TTY.hs

```diff
@@ -623,7 +623,9 @@ initialUiVmStateForTest
   -> IO UiVmState
 initialUiVmStateForTest opts@UnitTestOptions{..} (theContractName, theTestName) = do
   let state' = fromMaybe (error "Internal Error: missing smtState") smtState
-  (buf, len) <- flip runReaderT state' $ SBV.runQueryT $ symCalldata theTestName types []
+  (buf, len) <- case test of
```

Comment:
Can't we just create this in the `SymbolicTest` branche of the `case test of` below? I don't think it's used anywere else?

Reply:
unfortunately not because that is a `Stepper` type and not Symbolic. We could generalize EVMIO to be a `EVMSymbolic` or smth like that, but this felt like the easiest fix right now